timesheet_automatic_remind/README.rst

```diff
@@ -0,0 +1,64 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==========================
+Timesheet Automatic Remind
+==========================
+
+
+Develop the automatic emails to send every morning to people who
+didn't input their TMS (and to the person CC his manager at noon)
+
+
+Installation
+============
+
+To install this module, you need to:
+
+ * have basic modules installed (hr_timesheet, project_timesheet, project_issue_sheet)
+
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/Elico-Corp/odoo/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed feedback
+`here <https://github.com/Elico-Corp/odoo/issues/new?body=module:%20timesheet_activity_report%0Aversion:%20{8.0}%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
```

Comment:
remove this part following newest template (https://github.com/Elico-Corp/template-doc/blob/master/12._DEV_(Development_templates)/README.rst